tests/pyincore/test_hazardservice.py

```diff
@@ -475,6 +481,7 @@ def test_post_hurricanewf_hazard_values(hazardsvc):
 
 @pytest.mark.skip(reason="performance issues")
 def test_get_hurricanewf_json(hazardsvc):
+    # tested by ywkim on Feb 24 2023 and worked okay
```

Comment:
I dont think you need to add those inline comments? Thoughts @navarroc

Reply:
Yes, let's remove those since they aren't necessary. The merge of this branch will act as the metadata that says they were tested and by who.